res/img/icon-email-pill-avatar.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am 95% confident this SVG can be much simpler (likely the filter and ~5 transforms can go away) if exported in a different way. @nadonomy should be able to make it happen.

Reply:
This was the SVG supplied by Nad, though shoved through inkscape to fix some viewport problems.

Reply:
and by shoved through inkscape I mean in a later PR it is shoved through inkscape.

Reply:
Have you tried mentioning the filter and transform layers to him? In past, I believe he's been able to resolve it.

Reply:
Nope, I generally assume that the SVG is production ready from wherever I get it from. Will ask.

Reply:
I'm going to leave the email avatar for https://github.com/matrix-org/matrix-react-sdk/pull/3819 just so I don't have merge conflicts across my own PRs.